### PR TITLE
Edit docs

### LINKED ISSUE #12: Change readme.dev & contributing to link to TulipaEnergyModel
These common docs we just want to keep/update in 1 place.
Say something like "We follow the same guidelines as our main repo, TulipaEnergyModel(url)."

### PR DESCRIPTION
**Pull request by gnawin**
Give links to contributing and developer guidelines.

Closes #12

### CODE REVIEW

**Comment by gnawin** [comment]
@abelsiqueira I successfully built the docs locally, but something is wrong online. Any ideas?

File: docs/src/contributing.md [hunk 1]
```diff
@@ -1,26 +1,5 @@
...
-If the solution to the issue is clear, you can immediately create a pull request (see below).
-Otherwise, say what your proposed solution is and wait for a discussion around it.
-
-> **Tip**
->
-> Feel free to ping us after a few days if there are no responses.
-
-If your solution involves code (or something that requires running the package locally), check the [developer documentation](developer.md).
-Otherwise, you can use the GitHub interface directly to create your pull request.
+Please read these guidelines [CONTRIBUTING.md] (<https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md>) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```
Comment (abelsiqueira) [inline docs/src/contributing.md:5]:
```suggestion
Please read these guidelines [CONTRIBUTING.md] (https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```

File: docs/src/developer.md [hunk 2]
```diff
@@ -1,114 +1,5 @@
...
-Then, to build the documentation, run
-
-```julia
-julia> include("make.jl")
-```
-
-If you intend to rerun the build step, ensure you have the package `Revise` installed in your global environment, and run `using Revise` before including `make.jl`. Alternatively, close `julia` and reopen it.
-
-After building, the documentation will be available in the folder `docs/build/`. Open the `index.html` file on the browser to see it.
+Please read these guidelines [CONTRIBUTING.md] (<https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md>) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```
Comment (abelsiqueira) [inline docs/src/developer.md:5]:
```suggestion
Please read these guidelines [CONTRIBUTING.md] (https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```

**Review by abelsiqueira — DISMISSED** [review]

File: docs/src/contributing.md [hunk 3]
```diff
@@ -1,26 +1,5 @@
...
-If the solution to the issue is clear, you can immediately create a pull request (see below).
-Otherwise, say what your proposed solution is and wait for a discussion around it.
-
-> **Tip**
->
-> Feel free to ping us after a few days if there are no responses.
-
-If your solution involves code (or something that requires running the package locally), check the [developer documentation](developer.md).
-Otherwise, you can use the GitHub interface directly to create your pull request.
+Please read these guidelines [CONTRIBUTING.md](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```
Comment (clizbe) [inline docs/src/contributing.md:5]:
```suggestion
We follow the same guidelines as on our main repository, [TulipaEnergyModel](https://github.com/TulipaEnergy/TulipaEnergyModel.jl).
Please read these guidelines [CONTRIBUTING.md](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```

Comment (clizbe) [reply docs/src/contributing.md:5]: Also can you remove these files and just put this statement in the README.md?

**Review by clizbe — CHANGES_REQUESTED** [review]
Looks better, but can be even simpler!

File: docs/src/contributing.md (hunk 3 above)
Comment (gnawin) [reply docs/src/contributing.md]: Thanks. I made the changes as requested.

**Review by clizbe — APPROVED** [review]
LGTM!

**Pull request merged**